README.md

```diff
@@ -18,7 +18,7 @@ Our full-stack web development curriculum is completely free and self-paced. We
 * [Reporting Bugs and Issues](#reporting-bugs-and-issues)
 * [Reporting Security Issues](#reporting-security-issues)
 * [Contributing](#contributing)
-* [Platform, Build and Deployment Status](#platform-build-and-deployment-status)
+* [The platform, Build and Deployment Status](#platform-build-and-deployment-status)
```

Comment:
```suggestion
* [Platform, Build and Deployment Status](#platform-build-and-deployment-status)
```

This should be understood as platform status, build status and deployment status as three things.